src/scripts/journal.js

```diff
@@ -35,14 +35,14 @@ const content = document.querySelector("#content")
 const mood = document.querySelector("#mood")
 
 // Factory function that creates a new journal entry by taking each input above as an argument, and returns the value of those inputs when invoked.
-const createJournalEntry = (date, concept, content, mood) => {
-    return {
-        "date": date.value,
-        "concept": concept.value,
-        "content": content.value,
-        "mood": mood.value
-    }
-}
+const createJournalEntry = (date, concept, content, mood) => (
```

Comment:
Just another example of an implicit return... another added benefit is that we have to write less keystrokes to do the exact same thing.